about.css

```diff
@@ -0,0 +1,1149 @@
+  width: 70px;
+  z-index: -1;
+  top: -12%;
+  left: -20%;
+}
+
+.avatar {
+  display: block;
+  width: 130px;
+  height: 160px;
+}
+
+.FeauredSpeakers {
+  display: flex;
+  justify-content: center;
+  margin-top: 6rem;
+}
+
+hr {
+  width: 100px;
+  color: #e53d3d;
+  height: 7px;
+  margin-left: 150px;
+}
+
+.hidde {
+  display: none;
+}
+
+/* feature-speaker end  */
+
+/* partner section */
+
+#partner {
+  background-color: #515356;
+  padding: 2rem;
+}
+
+.partner-heading {
+  text-align: center;
+  color: wheat;
+}
+
+.partners-logos-container {
+  margin: 1rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  margin-top: 2rem;
+}
+
+.partners-logo {
+  width: 100px;
+}
+
+/* .partner section end  */
+
+/* footer section */
+#footer {
+  padding: 2rem;
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+}
+
+.left-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-img {
+  width: 110px;
+  height: 45px;
+  margin-right: 0.5rem;
+  margin-top: 2rem;
+}
+
+.footer-heading,
+.footer-description {
+  margin-left: 1rem;
+}
+
+.footer-heading {
+  font-size: 0.8rem;
+  color: #2b2b2b;
+}
+
+.footer-description {
+  font-size: 0.7rem;
+  color: #090909;
+}
+
+/* about page */
+
+.about-heading {
+  text-align: center;
+  padding: 2rem 0 0 1rem;
+  width: 80%;
+  color: #ec5242;
+}
+
+.about-description {
+  margin: 2.5rem auto 0;
+  background-color: rgb(212, 209, 209);
+  padding: 6%;
+  border: 1px solid var(--border-lines-color);
+}
+
+.contact-text {
+  background-color: rgb(212, 209, 209);
+  text-align: center;
+  font-size: 1.1rem;
+  padding: 1.5rem 2rem;
+  color: var(--main-text-color);
+}
+
+.email-contact {
+  background-color: rgb(212, 209, 209);
+  font-size: 1rem;
+  text-decoration: underline;
+  text-align: center;
+}
+
+#logo-section {
+  background-color: #fff;
+  padding: 2rem 0;
+}
+
+.logo-heading,
+.logo-description {
+  text-align: center;
+  color: var(--main-text-color);
+  padding: 0.5rem 1rem;
+}
+
+.logo-heading {
+  margin: 0 auto;
+  max-width: 80%;
+}
+
+.logo-description {
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+}
+
+.logo-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 0.5px solid var(--border-lines-color);
+  width: 60%;
+  padding: 1rem 2rem;
+  margin: 2rem auto;
+}
+
+#about {
+  background-color: #d3d3d3;
+  padding-bottom: 1rem;
+}
+
+.about-logo-img {
+  width: 150px;
+  height: auto;
+}
+
+.about-logo-text {
+  font-size: 0.9rem;
+  font-weight: 900;
+  width: 190px;
+}
+
+.date-location {
+  font-size: 0.5rem;
+  margin: 0.3rem 0;
+  border-top: 2px solid var(--border-lines-color);
+  border-bottom: 2px solid var(--border-lines-color);
+  text-align: left;
+  color: var(--main-text-color);
+}
+
+/******* */
+
+#upcomming {
+  padding-top: 2rem;
+  text-align: center;
+}
+
+.upcomming-heading {
+  color: var(--main-text-color);
+}
```

Comment:
- [ ] Please kindly make sure there is a divider after the title 😉

Your design
<img width="544" alt="Screen Shot 2023-01-14 at 04 31 37" src="https://user-images.githubusercontent.com/74501523/212444490-4694c0f3-c186-4829-9980-6eefa5a5694e.png">

Original design
<img width="544" alt="Screen Shot 2023-01-14 at 04 31 59" src="https://user-images.githubusercontent.com/74501523/212444507-113b3d9e-f8c1-43a1-9de4-30b26a36c29f.png">